### PR TITLE
Fix build for missing last_modified_at tag

### PR DESCRIPTION
**Pull request by SpamapS**
I was mistaken thinking this plugin was available in GH pages, but it is
not.

### CODE REVIEW

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/bonnyci.org/29/1487376795.71/bonnyci-run-check : FAILURE in 2m 16s

File: beta/README.md [hunk 1]
```diff
@@ -186,9 +186,7 @@ space for future updates. If you want to join our closed beta, you can
 email me personally (see below) or join us on Freenode IRC in #BonnyCI.
 
 *Clint Byrum is a Cloud Architect at IBM, you can contact him
-via Twitter: @SpamapS, or email [clint at fewbar dot com](mailto:clint at fewbar dot com)
-
-Last Updated: {% last_modified_at %B %d, %Y %}*
+via Twitter: @SpamapS, or email [clint at fewbar dot com](mailto:clint at fewbar dot com)*
```
Comment (mlangbehn) [inline beta/README.md:189]: Trailing asterix.

**Review by mlangbehn — CHANGES_REQUESTED** [review]

File: beta/README.md (hunk 1 above)
Comment (mlangbehn) [reply beta/README.md:189]: ^ Please disregard. I missed the opening asterix on 188. ^

**Review by mlangbehn — APPROVED** [review]
Looks good to me.

**Pull request merged**